.github/workflows/ci.yml

```diff
@@ -92,6 +86,11 @@ jobs:
           mkdir -p ~/optee-qemuv8 && cd ~/optee-qemuv8 &&
           repo init -u https://github.com/OP-TEE/manifest.git -m qemu_v8.xml &&
           repo sync -j4 --no-clone-bundle
+      - name: Setting up $HOME
+        run: |
+          cp /root/.bashrc $HOME/.bashrc &&
+          ln -sf /root/.rustup ~/.rustup &&
+          ln -sf /root/.cargo ~/.cargo
```

Comment:
Seems that CI works fine without setting that environment in previous running jobs, so it's really needed to do this?

Reply:
I think it's because we unconditionally installed our own version of rustup from https://rustup.rs/ before. My changes to `setup.py` make us use an installed version if it exists, though, which means that installed version has to work. And the version in the GitHub Actions container we use doesn't seem to work unless those symlinks exist.